hastory-server/src/Data/Hastory/Server.hs

```diff
@@ -98,21 +131,43 @@ generateToken = T.pack . take tokenLength . randomRs ('a', 'z') <$> liftIO newSt
 reportPort :: MonadLogger m => Options -> m ()
 reportPort Options {..} = logInfo $ "Starting server on port " <> T.pack (show _oPort)
 
--- | Logs information about the data file. This data file serves as a database for this primitive append-only server.
-reportDataFileStatus :: (MonadIO m, MonadLogger m) => Options -> m ()
-reportDataFileStatus Options {..} = do
-  dataFileExists <- liftIO $ fileExist _oDataOutputFilePath
-  if dataFileExists
-    then logInfo $
-         "Data file exists at " <> T.pack _oDataOutputFilePath <> ". Appending commands to it."
-    else logInfo "Data file doesn't exist. Creating a new one."
+-- | We use a hardcoded file name for database. Path to database is chosen by the user.
+mkDbFile :: FilePath -> FilePath
+mkDbFile dir = dir </> "hastory.db"
+
+-- | We use a hardcoded file name for file data storage. Path to database is chosen by the user.
+mkStorageFile :: FilePath -> FilePath
+mkStorageFile = (</> "data.txt")
+
+-- | Creates directory if it cannot be found. Only difference between this
```

Comment:
I don't think this logging brings us anything.
We can just use `ensureDir`: https://hackage.haskell.org/package/path-io-1.6.0/docs/Path-IO.html#v:ensureDir